### PR TITLE
Add captions to plots inside `textOutput()`

### LINKED ISSUE #6: Plot Captions
Include plot captions below the plots (inside  `nav_panel()`) as a `textOutput()`

### PR DESCRIPTION
**Pull request by howardbaik**
#### What changes are being implemented in this Pull Request?

I added captions below all the plots on this dashboard.

#### What was your approach?

Used regular text to put captions below the plots. In terms of Shiny code, I used functions listed in https://mastering-shiny.org/basic-ui.html?q=text#text. 

#### What GitHub issue does your pull request address?

https://github.com/FredHutch/itn-dashboard/issues/6

### Tell potential reviewers what kind of feedback you are soliciting.

Whether the captions make sense.

### CODE REVIEW

**Comment by cansavvy** [comment]
@howardbaek I'm not exactly sure the correct way to resolve the file conflict here, can you take a look?

File: app.R [hunk 1]
```diff
@@ -431,7 +466,10 @@ server <- function(input, output) {
       ylim(c(0, 4200)) + 
       facet_wrap(~target_audience) + 
       scale_fill_manual(values=cbPalette)
-    
+  })
+  # Caption
+  output$learner_by_modality_caption <- renderText({
+    "Level of engagement in courses by modality, segmented by different target audience categories—Everyone, Leadership, New to data, and Software developers. It contrasts the number of visitors/enrollees across website learners and total course enrollments."
```
Comment (cansavvy) [inline app.R]:
```suggestion
    "Level of engagement in courses by modality, segmented by different target audience categories: everyone, leadership, new to data, and software developers. It contrasts the number of visitors/enrollees across website learners and total course enrollments."
```

File: app.R [hunk 2]
```diff
@@ -474,6 +516,11 @@ server <- function(input, output) {
       theme(legend.position = c(0.9, 0.85),
             text = element_text(size = 17, family = "Arial"))
   })
+  # Caption
+  output$coursera_learner_caption <- renderText({
+    "Number of enrollments for various Coursera courses, delineating the popularity of courses among different target audiences, such as leadership, new to data, and software developers, with the 'Computing for Cancer Informatics' course showing the highest enrollment figures."
```
Comment (cansavvy) [inline app.R:520]: I like the added interpretation however, we might want to be careful about calling out particular results like this unless we are prepared to keep these up to date. In other words, if another course catches up to Computing for Cancer Informatics, we'd have to come back and change this. Computing for Cancer Informatics is one of the oldest course on Coursera so the other courses could catch up given enough time.

File: app.R [hunk 3]
```diff
@@ -492,6 +539,10 @@ server <- function(input, output) {
       theme(legend.position = c(0.9, 0.85),
             text = element_text(size = 17, family = "Arial"))
   })
+  # Caption
+  output$leanpub_learner_caption <- renderText({
+    "Number of Leanpub course enrollments, highlighting interest across different subjects with 'Leadership in Cancer Informatics' attracting the most learners, categorized by target audiences like leadership, those new to data, and software developers."
```
Comment (cansavvy) [inline app.R:543]: Same comment here! I like it but this might change 😄

File: app.R [hunk 4]
```diff
@@ -570,6 +638,10 @@ server <- function(input, output) {
             text = element_text(size = 17, family = "Arial")) +
       scale_fill_manual(values = c("#440154", "#28ae80"))
   })
+  # Caption
+  output$workshop_career_stage_caption <- renderText({
+    "Distribution of workshop registrants across different career stages, with the majority being PhD students and postdocs, and a distinction between trainees and non-trainees indicated by colo"
```
Comment (cansavvy) [inline app.R]:
```suggestion
    "Distribution of workshop registrants across different career stages, with the majority being PhD students and postdocs, and a distinction between trainees and non-trainees indicated by color."
```

File: app.R [hunk 5]
```diff
@@ -580,6 +652,10 @@ server <- function(input, output) {
                          colors = c("#98fb98", "#83D475", "#355E3B"),
                          min.freq = 3, scale = c(3, .4))
   })
+  # Caption
+  output$like_most_about_workshop_caption <- renderText({
+    "Terms like 'interactive', 'hands-on', 'activity', and 'how' suggesting a focus on practical engagement and application, as well as the importance of the workshop's format and content delivery."
```
Comment (cansavvy) [inline app.R]:
```suggestion
    "This word cloud shows the most used terms in workshop reviews when asked 'what did you like most about the workshop?' Terms like 'interactive', 'hands-on', 'activity', and 'how' suggests workshop participants really enjoy the interactivity of the workshops."
```

File: app.R [hunk 6]
```diff
@@ -589,7 +665,31 @@ server <- function(input, output) {
                          colors = c("#98fb98", "#83D475", "#355E3B"),
                          min.freq = 3, scale = c(4, .4))
   })
-  
+  # Caption
+  output$recommendation_improvement_caption <- renderText({
+    "Feedback themes for a workshop, with the most significant focus on 'more', 'how', 'time', and 'example', suggesting participants are interested in detailed explanations, practical examples, and perhaps more time allocated for these activities."
```
Comment (cansavvy) [inline app.R]:
```suggestion
    "Feedback themes for a workshop, with the most significant focus on 'more', 'how', 'time', and 'example', suggesting participants are interested in detailed explanations, practical examples, and perhaps more time or more workshop offerings. "
```

File: app.R [hunk 7]
```diff
@@ -589,7 +665,31 @@ server <- function(input, output) {
...
+            legend.position="none", 
+            text = element_text(size = 17, family = "Arial"),
+            plot.margin = unit(c(.75,.5,.5,.5), "cm")) + 
+      xlab(NULL) +
+      ylab("N") +
+      ggtitle("Collaborations with All Individuals")
+  })
+  # Caption
+  output$collaboration_all_caption <- renderText({
+    "Distribution of various academic and professional collaborations, such as manuscript authorship, course development, and networking events, showing how frequently each collaboration was completed."
```
Comment (cansavvy) [inline app.R]:
```suggestion
    "This plot shows the large variety of collaborations and engagements with the ITN. Distribution of various academic and professional collaborations, such as manuscript authorship, course development, and networking events, showing how frequently each collaboration was completed."
```

File: app.R [hunk 8]
```diff
@@ -608,25 +708,13 @@ server <- function(input, output) {
...
-            legend.position="none", 
-            text = element_text(size = 17, family = "Arial"),
-            plot.margin = unit(c(.75,.5,.5,.5), "cm")) + 
-      xlab(NULL) +
-      ylab("N") +
-      ggtitle("Collaborations with All Individuals")
-    
+  # Caption
+  output$collaboration_itcr_caption <- renderText({
+    "The number of individuals who have helped with each type of collaboration within the ITCR community. Most individuals were ITCR investigators but some were trainees or scientific staff."
```
Comment (cansavvy) [inline app.R]:
```suggestion
    "This plot also shows the number of collaborations but only collaborations that were completed with ITCR funded individuals. The number of individuals who have helped with each type of collaboration within the ITCR community. Most individuals were ITCR investigators but some were trainees or scientific staff."
```

**Review by cansavvy — APPROVED** [review]
This is great @howardbaek Thanks for adding these descriptions -- I wouldn't have been sure how to do it seamlessly! I just have some minor suggestions for wording changes. Thanks again!

I didn't run this locally to see the preview, I'm just kinda trusting it looks okay. In a later PR we'll need to figure out how to do previews as well as fix the overlapping figure keys. (I'll make an issue for this if there is not yet). 

Thanks for your work on this @howardbaek

File: app.R (hunk 3 above)
Comment (howardbaik) [reply app.R:543]: Yeah good point. I'll get rid of sentences like this in all the other captions.

**Pull request merged**